src/sweetalert2.scss

```diff
@@ -613,13 +613,16 @@ body {
   margin: $swal2-icon-margin;
   border: .25rem solid transparent;
```

Comment:
let's replace `.25rem` with `.25em` to make the icon border relative to the modal base font-size.